tests/lib/rules/prefer-global/buffer.js

```diff
@@ -7,41 +7,47 @@
 const RuleTester = require("#test-helpers").RuleTester
 const rule = require("../../../../lib/rules/prefer-global/buffer")
 
+const cjsMethods = [`require`, "process.getBuiltinModule"]
```

Comment:
The variable name is bad.. Also inconsistent quote style in these two function names..

Reply:
Thank you for your comment! I fixed them.